projects/test-app/src/hslayers-app/hslayers-app.component.ts

```diff
@@ -392,7 +392,7 @@ export class HslayersAppComponent {
             },
           ],
           proxyPrefix: window.location.hostname.includes('localhost')
-            ? `${window.location.protocol}//${window.location.hostname}:8085/`
+            ? `${window.location.protocol}//127.0.0.1:8085/`
```

Comment:
commited intentionally, @FilipLeitner ?

Reply:
Yes, kinda intentional. Not sure what changed but when running e2e cypress was not able to access proxy under the 'localhost' hostname. If I remember correctly both 127.0.0.1 and 0.0.0.0 was working ...just localhost not. Tried few cypress setting but no success. 
And its pretty much the same thing so I felt its safe to be changed in this way

Reply:
Ack. Verified. Happens to me as well. So let's go this way now. Thanks for explaining.